smui/Dockerfile

```diff
@@ -1,12 +1,10 @@
 FROM querqy/smui:3.14.0
 
-COPY conf/smui2solrcloud.sh /smui/conf/smui2solrcloud.sh
-COPY conf/*.py /smui/conf/
-COPY conf/predefined_tags.json /smui/conf/predefined_tags.json
+COPY --chown=1024:1024 conf/* conf/
```

Comment:
I notice below a `USER smui`, could this line be `--chown=smui:smui conf/* conf/`????

Reply:
> I notice below a `USER smui`, could this line be `--chown=smui:smui conf/* conf/`????

probably; should be the same... I'll check and send a commit...